Web_Development/Real-TimeMulti-UserWhiteboard/app.py

```diff
@@ -0,0 +1,21 @@
+
+### 2. `app.py`
+
+```python
```

Comment:
Hey @shubhagarwal1 ,
Please remove this, it is showing error. Otherwise program works fine, will approve once updated.
Thanks!